public/index.html

```diff
@@ -20,18 +21,205 @@
+        <aside class="friends">
+            <div>
+                <span class="friends__text">Лента друзей </span>
+                <img src="%PUBLIC_URL%/images/friend.png" width="20" height="20">
+            </div>
+            <hr class="hr__color">
+            <ul class="friends__list">
+                <li>
+                    <img src="%PUBLIC_URL%/images/avatar1.jpg" width="64" height="64" align="left" class="avatar">
+                    <ul class="friends__track">
+                        <li> Иван Иванов</li>
+                        <li> DADDY-PSY</li>
+                        <li>PSY</li>
+                    </ul>
+                </li>
+                <br>
+                <li>
+                    <img src="%PUBLIC_URL%/images/avatar2.jpg" width="64" height="64" align="left" class="avatar">
+                    <ul class="friends__track">
+                        <li> Павел Павлов</li>
+                        <li> Donda-Hurricane</li>
+                        <li>Kanye West</li>
+                    </ul>
+                </li>
+                <br>
+                <li>
+                    <img src="%PUBLIC_URL%/images/avatar3.jpg" width="64" height="64" align="left" class="avatar">
+                    <ul class="friends__track">
+                        <li> Андрей Андреев</li>
+                        <li> Dystopia-Mafia</li>
+                        <li>Travis Scott</li>
+                    </ul>
+                </li>
+            </ul>
+        </aside>
+        <footer class="footer">
+           <div class="footer__cover">
+               <img src="%PUBLIC_URL%/images/cover.jpg" width="64" height="64" align="left" class="footer__avatar" >
+                    <ul class="footer__track">
+                        <li class="track__name"> STARGAZING</li>
+                        <li class="track__musician"> Travis Scott</li>
+                    </ul>
+               <img class="like__button" src="%PUBLIC_URL%/images/like.png" width="16" height="16">
+           </div>
+            <div>
+                <ul class="footer__buttons">
+                    <li class="footer__play">
+                        <img class="footer__button" src="%PUBLIC_URL%/images/shuffle.png" width="20px" height="20px">
+                        <img class="footer__button" src="%PUBLIC_URL%/images/play-back.png" width="20px" height="20px">
+                        <img class="footer__button" src="%PUBLIC_URL%/images/play-button.png" width="25px" height="25px">
+                        <img class="footer__button" src="%PUBLIC_URL%/images/play-forward.png" width="20px" height="20px">
+                        <img class="footer__button" src="%PUBLIC_URL%/images/replay.png" width="20px" height="20px">
+                    </li>
+                    <li class="footer__timecode">
+                        <div>
+                            <span class="footer__time">2:34</span>
+                            <img class="time" src="/images/time.png">
```

Comment:
Тут не хватает "%PUBLIC_URL%/
```suggestion
                            <img class="time" src=""%PUBLIC_URL%/images/time.png">
```